tests/spread/integration/apache2/task.yaml

```diff
@@ -0,0 +1,26 @@
+summary: Integration tests for apache2
+
+execute: |
+  rootfs="$(install-slices apache2_bins base-passwd_data)"
+
+  mkdir -p "${rootfs}/dev"
+  mount --rbind /dev "${rootfs}/dev"
+  echo "ServerName localhost" >> "${rootfs}/etc/apache2/apache2.conf"
+  test "$(chroot "${rootfs}" a2query -M)" = "event"
+  # Enabling mpm_prefork since it causes a coredump and can't locate libgcc_s.so.1
+  # AH00051: child pid 1890 exit signal Abort (6), possible coredump in /etc/apache2
+  # libgcc_s.so.1 must be installed for pthread_exit to work
```

Comment:
I think we should clarify in this comment that it is `mpm_event` that causes a coredump.